### PR TITLE
Bulk update modal

### LINKED ISSUE #8383: [CAL-1529] Fix bulk update of default location
Darkmode looks really bad here 

<sub>[CAL-1529](https://linear.app/calcom/issue/CAL-1529/fix-bulk-update-of-default-location)</sub>

### PR DESCRIPTION
**Pull request by sean-brydon**
Use our checkbox component in bulk update modal to ensure it fits with the styles of the rest of the app.

Also supports darkmode :) 

Screenshot 2023-04-21 at 09 22 18

### CODE REVIEW

**Review by joeauyeung — APPROVED** [review]
LGTM

File: packages/features/eventtypes/components/BulkEditDefaultConferencingModal.tsx [hunk 1]
```diff
@@ -33,7 +33,7 @@ export function BulkEditDefaultConferencingModal(props: { open: boolean; setOpen
   if (isFetching || !open || !data?.eventTypes) return null;
 
   return (
-    <Dialog name="Bulk Default Location Update" open={props.open} onOpenChange={props.setOpen}>
+    <Dialog name="Bulk Default Location Update" open={true} onOpenChange={props.setOpen}>
```
Comment (emrysal) [inline packages/features/eventtypes/components/BulkEditDefaultConferencingModal.tsx:36]: ?.?

**Review by emrysal — CHANGES_REQUESTED** [review]

File: packages/features/eventtypes/components/BulkEditDefaultConferencingModal.tsx (hunk 1 above)
Comment (sean-brydon) [inline packages/features/eventtypes/components/BulkEditDefaultConferencingModal.tsx:36]:
```suggestion
    <Dialog name="Bulk Default Location Update" open={props.open} onOpenChange={props.setOpen}>
```

Comment (sean-brydon) [reply packages/features/eventtypes/components/BulkEditDefaultConferencingModal.tsx]: Fixed dw dw

**Review by emrysal — APPROVED** [review]

**Pull request merged**